### PR TITLE
Providing explanatory help text for configuration file

### LINKED ISSUE #79: Document thresholds and valid objects
The thresholds here
https://github.com/CityOfLosAngeles/automated-walk-bike-counter/blob/a4c95d7cc23ed48df6c9a286bc93dcf54f83e517/automated_walk_bike_counter/core/configuration.py#L32-L124
are do not have very well documented helpstrings, largely because I don't know what they do. @vahedi25411, can you add some more descriptive help strings to these configurables?

Likewise, I still don't fully understand the difference between `VALID_OBJECTS` and `search_objects`. Can you add some more text describing the difference? Should `VALID_OBJECTS` even be user-configurable, or should we remove it?

### PR DESCRIPTION
**Pull request by vahedi25411**
Fixes #79

### CODE REVIEW

File: automated_walk_bike_counter/core/configuration.py [hunk 1]
```diff
@@ -32,23 +32,26 @@ def list_of_items(arg):
...
+    " to assign those objects together",
     required=False,
     default=90,
     type=float,
     env_var="PED_COST_THRESHOLD",
 )
 parser.add_argument(
     "--bus_cost_threshold",
-    help="Bus cost threshold",
+    help="Maximum distance between two pedestrian objects in two frames that we can use"
```
Comment (ian-r-rose) [inline automated_walk_bike_counter/core/configuration.py:44]: Should say "bus"

File: automated_walk_bike_counter/core/configuration.py [hunk 2]
```diff
@@ -32,23 +32,26 @@ def list_of_items(arg):
...
+    " to assign those objects together",
     default=110,
     required=False,
     type=float,
     env_var="BUS_COST_THRESHOLD",
 )
 parser.add_argument(
     "--truck_cost_threshold",
-    help="Truck cost threshold",
+    help="Maximum distance between two pedestrian objects in two frames that we can use"
```
Comment (ian-r-rose) [inline automated_walk_bike_counter/core/configuration.py:53]: Should say "Truck"

File: automated_walk_bike_counter/core/configuration.py [hunk 3]
```diff
@@ -183,19 +192,23 @@ def list_of_items(arg):
...
+    help="The colors that we want to use to highlight the objects, that we have "
+    "in search_objects, in the output",
 )
 
 
 parser.add_argument(
-    "--aoi", type=list_of_tuples, help="The list of aoi coordinates.",
+    "--aoi",
+    type=list_of_tuples,
+    help="The list of coordinates that are used to" " indicate the area of interest.",
```
Comment (ian-r-rose) [inline automated_walk_bike_counter/core/configuration.py:211]: We can concatenate these two strings into one.

File: automated_walk_bike_counter/core/configuration.py (hunk 1 above)
Comment (vahedi25411) [reply automated_walk_bike_counter/core/configuration.py]: Thanks, @ian-r-rose, Sorry for the mistake

**Comment by vahedi25411** [comment]
@ian-r-rose Could you please check it again?

**Pull request merged**